<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>693</width>
    <height>426</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="imgLabel1">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>50</y>
     <width>361</width>
     <height>331</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>-10</y>
     <width>251</width>
     <height>431</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="loadButton">
      <property name="text">
       <string>load image</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="trainButton">
      <property name="text">
       <string>Train Database</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="segmentButton">
      <property name="text">
       <string>Segment</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="recogniseButton">
      <property name="text">
       <string>Recognization</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
